src/ActionList/index.ts

```diff
@@ -18,8 +18,8 @@ export type {ActionListLeadingVisualProps, ActionListTrailingVisualProps} from '
  * Collection of list-related components.
  */
 export const ActionList = Object.assign(List, {
-  /** Collects related `Items` in an `ActionList`. */
-  Group,
```

Comment:
We want to keep this, right?

reference: https://github.com/primer/react/pull/2878/files#r1147703351

Reply:
Ah, yeah! I removed this when I thought we were deprecating it by moving the path.